Italian/finestra/apri.md

```diff
@@ -1,2 +1,8 @@
 
-Ti appare un paesaggio fiabesco
+Ti appare un paesaggio fiabesco e in lontananza sullo sfondo scorgi un gigantesco T-rex
+
+Come reagisci?
+
+[Prendi il telefono per fotografarlo] (../t-rex/mangiato.md)
+[Crei una corda col lenzuolo e scendi dalla finestra] (../t-rex/cadi.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.